core/arch/arm/tee/entry_std.c

```diff
@@ -76,6 +77,42 @@ static TEE_Result set_mem_param(const struct optee_msg_param *param,
 	return TEE_SUCCESS;
 }
 
+static TEE_Result set_sdp_shm_param(const struct optee_msg_param *param,
```

Comment:
This function and `set_ns_shm_param()` could be simplified using a helper function:
```
static TEE_Result mem_param_from_param_mobj(struct param_mem *mem,
                                            const struct optee_msg_param *param,
                                            struct mobj *mobj)
```
Instead